index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="chrome=1">
+  <title>VIKING GARAGE Motorcycle Riders global marketplace: rentals, repairs</title>
+  <meta name="Description" content="A worldwide marketplace and community for real people riding real motorcycles. Local private rentals, repairs, tours. VG brings the Platform and logistics." />
+  <meta name="Keywords" content="motorcycle, rental, repair, tour, marketplace, community, global, local, private, worldwide, motocross, enduro, cafe racer, scrambler, chopper, bobber, brat style, adventure, braap" />
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,200,300,400|Loved+by+the+King" rel="stylesheet">
+  <link href="/assets/css/custom.css" rel="stylesheet">
+  <script src="https://use.fontawesome.com/8ab56a8e28.js"></script>
```

Comment:
It's AwesomeFonts politics: you can't just get CDN link, you have to give your email and they send you this script, more here: http://fontawesome.io/get-started/